v63003/outref/gtm8182.txt

```diff
@@ -68,7 +68,7 @@ Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_P
 # Run gtm8182.m again
     Update the INST1 DB
     Update the INST3 DB with an extended reference
-##TEST_AWK^%YDB-E-REPLINSTMISMTCH.*
+##TEST_AWK%YDB-E-REPLINSTMISMTCH, Process has replication instance file ##TEST_PATH##/mumps.repl \(jnlpool shmid = [0-9]*\) open but database .*/instance3/mumps.dat is bound to instance file .*/instance3/mumps.repl \(jnlpool shmid = [0-9]*\)
```

Comment:
Can we use a ##REMOTE_TEST_PATH## instead of the .*
This comments applies to all the error message TEST_AWKs.